com.pot_app.pot.metainfo.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.pot_app.pot</id>
+  
+  <name>Pot</name>
+  <summary>Cross-platform translation software</summary>
+  <developer_name>Pylogmon</developer_name>
+
+  <url type="homepage">https://pot-app.com/</url>
+  <url type="bugtracker">https://github.com/pot-app/pot-desktop/issues</url>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-3.0-only</project_license>
+  
+  <description>
+    <p>A cross-platform software for text translation and recognize.</p>
+    <p xml:lang="zh_CN">一个跨平台的划词翻译和 OCR 软件</p>
+  </description>
+  
+  <launchable type="desktop-id">com.pot_app.pot.desktop</launchable>
```

Comment:
> com.pot-app.desktop

That's wrong, the one here is correct.

> [...]replace any hyphens in the ID with underscores[...]

https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic

Reply:
But, `com.pot-app.desktop` is not a valid app id

Reply:
no. it MUST match with the app-id and you can't have `-` there (to be DBus compliant) so they are replaced by `_`. and you should too upstream. (`_` happen to be invalid in domains so it's safe)

Reply:
ok i don't know flathub's rules for this but i mean is it ok to write `com.pot_app.desktop`?

Reply:
> com.pot_app.desktop

No https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html#desktop-file-id

> The name of the desktop entry should follow the "reverse DNS" convention: it should start with a reversed DNS domain name controlled by the author of the application, in lower case. The domain name should be followed by the name of the application

Reply:
no it's not.

`.desktop` is the suffix of the desktop file. That mean `com.pot_app` would be the id but it's too short. It's missing the app name. Also I will reiterate that it MUST match the app-id. 

https://docs.flatpak.org/en/latest/conventions.html#application-ids :

> The format is in reverse-DNS style so the first section should be a domain controlled by the project and the trailing section represents the specific project.

Reply:
so I should close this pr and open a new pr that it's app id is `com.pot_app.desktop`

Reply:
So that's it. Thank you for your reply, but I'm still confused. In this case, is it better to write `com.pot_app.pot` directly, consistent with the title of this PR?

Reply:
> so I should close this pr and open a new pr that it's app id is `com.pot_app.desktop`

I think it's great.

Reply:
> so I should close this pr and open a new pr that it's app id is `com.pot_app.desktop`?

No your PR and the id is fine

Reply:
Oh, my God, I just discovered that this line is the name of the `desktop` file. I thought it was the appid written here.

Reply:
Maybe I should change the appid of the upstream project to `com.pot_app.pot`? I wrote `com.pot-app.desktop` so that I can develop 'com.pot-app.mobile' in the future, but now I don't know how to correctly name my appid.